packages/wallet/WalletManager.test.js

```diff
@@ -0,0 +1,191 @@
+         signTransaction: [Function: signTransaction],
+         sign: [Function: sign],
+         encrypt: [Function: encrypt] },
+      balance: '0' }
+  */
+  expectedAddress = '0x2d8Cce8A8B308a077Eb0e39331258c355c55d04e';
+  expectedPrivateKey = '0xb1cf5f0991e165de0e832bb3304846f0e902c0fdef39deece4c14f6625dc5a61';
+  account = await wallet.getAccount(5);// maximum 2^32 -1, min 0, must be integer
+  expect(account.getAddress()).toBe(expectedAddress);
+  expect(account.getPrivateKey()).toBe(expectedPrivateKey);
+});
+
+test('setAccountStellar', async () => {
+  jest.setTimeout(30000);// the default timeout is 5000ms
+
+  const wallet = new WalletManager('stellar');
+  wallet.setSeed(seedPhrases);
+  wallet.setWallet();
+  let expectedPrivateKey = 'SDJNCBWIH4GU377ICXYL7NEI5Z2GWOR2Y3PAQVI2HJHJ7MSB42PP4KVW';
+  let expectedPublicKey = 'GCDAFTYQTU2YVNPCJVIZ6IT2MKSL2KRY724ODR3Y5AJ5NZ2CD6Z7A7GO';
+  const expectedBalance = '10000.0000000';
+  let account = await wallet.getAccount();
+
+  expect(account.getPrivateKey()).toBe(expectedPrivateKey);
+  expect(account.getAddress()).toBe(expectedPublicKey);
+  expect(account.getBalance()).toBe(expectedBalance);
+
+  expectedPrivateKey = 'SCHDEOGWKZYYHDTCQFEXMQ3VVDCOZIUBVSJN4DGTOCB5FCCKBYQEG4PA';
+  expectedPublicKey = 'GCUTV7FC4GITQI6KJASMKH7WX3NTDLNUBHFZNXVJB4DJGGTUL6I7XAVT';
+  account = await wallet.getAccount(10);
+  expect(account.getPrivateKey()).toBe(expectedPrivateKey);
+  expect(account.getAddress()).toBe(expectedPublicKey);
+  expect(account.getBalance()).toBe(expectedBalance);
+  /*
+      Account {
+      network: 'stellar',
+      testAddress: 'https://horizon-testnet.stellar.org/accounts/',
+      account:
+       Keypair {
+         type: 'ed25519',
+         _secretSeed: <Buffer 8e 32 38 d6 56 71 83 8e 62 81 49 76 43 75 a8 c4 ec a2 81 ac
+         92 de 0c d3 70 83 d2 88 4a 0e 20 43>,
+         _secretKey: <Buffer 8e 32 38 d6 56 71 83 8e 62 81 49 76 43 75 a8 c4 ec a2 81 ac 92 de 0c
+          d3 70 83 d2 88 4a 0e 20 43 a9 3a fc a2 e1 91 38 23 ca48 24 c5 1f f6 be db 31 ad ... >,
+         _publicKey: <Buffer a9 3a fc a2 e1 91 38 23 ca 48 24 c5 1f f6 be db 31 ad b4 09 cb
+          96 de a9 0f 06 93 1a 74 5f 91 fb> },
+      balance: '10000.0000000' }
+  */
+});
+
+// setting inflation stellar
+
+test('encryptAndDecryptAccountETH', async () => {
+  const wallet = new WalletManager('ethereum');
+  wallet.setSeed(seedPhrases);
+  wallet.setWallet();
+  // encrypt hd wallet, use case
+  const account = await wallet.getAccount();
+  const encrypted = wallet.encrypt(account, password);
+  const decryptedAccount = wallet.decrypt(encrypted, password);
+  expect(decryptedAccount.getAddress()).toEqual(account.getAddress());
+  expect(decryptedAccount.getPrivateKey()).toEqual(account.getPrivateKey());
+  expect(decryptedAccount.getNetwork()).toEqual(account.getNetwork());
+});
+
+// test('encryptAndDecryptAccountStellar', async () => {
```

Comment:
Why is this case commented out?

Reply:
Previously commented out because I do not know why it fails. Now it is fixed and added back.